selfdrive/car/gm/carcontroller.py

```diff
@@ -13,8 +13,8 @@ def __init__(self, car_fingerprint):
     if car_fingerprint == CAR.VOLT:
       self.STEER_MAX = 300
       self.STEER_STEP = 2              # how often we update the steer cmd
-      self.STEER_DELTA_UP = 7          # ~0.75s time to peak torque (255/50hz/0.75s)
-      self.STEER_DELTA_DOWN = 17       # ~0.3s from peak torque to zero
+      self.STEER_DELTA_UP = 8          # ~0.75s time to peak torque (300/50hz/0.75s)
+      self.STEER_DELTA_DOWN = 20       # ~0.3s from peak torque to zero
```

Comment:
this requires a safety change in panda. Also, I would rather fix the comments instead of changing the rate limits. They were tuned for what was considered safe.

Reply:
I just thought the point was to hit peak torque in or under 0.75 seconds. What I'm seeing now is when we're hitting the max steering cmd of 300 actual delivered torque is still falling behind to around 250 ish and slowly creeps up if max steer remains saturated from what I can tell.  By raising the step from 7 to 8 this gets actual torque delivered to better match the steering cmd and stays within commas safety standards.

I did also submit the PR for the panda changes. 

So just to be clear you don't want to raise the steer delta to match time to peak torque now that max steer is now 300?

Reply:
@CryptoKylan , correct: the rate limit should stay the same, as the safety standard is defined in terms of rate of steer torque (~Nm/s) rather than how long it takes to get to peak torque. Makes sense?